### PR TITLE
Integration test gherkin story for automatic device provisioning

### PR DESCRIPTION
**Pull request by pcenov**
Adding a feature file and a scenario for https://github.com/learningequality/kolibri/issues/8350

### CODE REVIEW

**Comment by pcenov** [comment]
@radinamatic

File: integration_testing/features/super-admin/commands/super-admin-automatic-device-provisioning.feature [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        "superuser": {
+            "username": "superuser",
+            "password": "password"
+        }
+    }
+
+  Scenario: Setup a device by modifying the AUTOMATIC_PROVISION_FILE setting in the options.ini file
+    When I add "AUTOMATIC_PROVISION_FILE = <file path>" under the [Paths] section
+      And I save my changes
+      And I run the "kolibri start" command in the terminal or command prompt
```
Comment (radinamatic) [inline integration_testing/features/super-admin/commands/super-admin-automatic-device-provisioning.feature:38]: Does it display the output even without running the ``kolibri start`` command with the flag ``--foreground``?

Comment (rtibbles) [reply integration_testing/features/super-admin/commands/super-admin-automatic-device-provisioning.feature:38]: It probably won't, but the output would be in the logs, at least.

Comment (radinamatic) [reply integration_testing/features/super-admin/commands/super-admin-automatic-device-provisioning.feature:38]: I suspected as much, thank you!

**Review by radinamatic — APPROVED** [review]
Looking good, thank you!
Fixes #8350

**Pull request merged**